src/tests/cityCard.test.js

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import CityCard from '../components/CityCard';
+
+const fakeData = {
+  name: 'name', continent: 'continent', flag: 'flag', lat: 'lat', lon: 'lon',
+};
+
+test('should render CityCard', () => {
+  render(<CityCard props={fakeData} />);
+});
```

Comment:
- [ ] Good job for Create integration tests for your application using the React Testing Library. But Kindly makes sure that your test cases are passing the requirement , So kindly fix it.

![Capture](https://user-images.githubusercontent.com/93598189/226131620-655f755e-4da3-4bf2-beef-04a546097361.PNG)